### PR TITLE
pest: add page

### PR DESCRIPTION
**Pull request by owenvoke**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pest.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Run tests in the current directory:
+
+`pest`
+
+- Run tests annotated with the given group:
+
+`pest --group {{name}}`
+
+- Run tests and generate a coverage report in `stdout`:
```
Comment (navarroaxel) [inline pages/common/pest.md:18]: I'm not sure about this example description:
```suggestion
- Run tests and print the coverage report to `stdout`:
```

File: pages/common/pest.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# pest
+
+> A PHP testing framework with a focus on simplicity.
+> More information: <https://pestphp.com>.
+
+- Initialise a standard Pest configuration in the current directory:
```
Comment (navarroaxel) [inline pages/common/pest.md:6]: We have pages that say `local path`, I think we need to uniform the expression for `current directory`/`local path`:
```suggestion
- Initialise a standard Pest configuration in the local path:
```

File: pages/common/pest.md (hunk 1 above)
Comment (owenvoke) [reply pages/common/pest.md:18]: Whoops, yeah... it doesn't make sense. 🤦🏻

File: pages/common/pest.md (hunk 2 above)
Comment (owenvoke) [reply pages/common/pest.md:6]: Hmm, personally I feel that current directory makes more sense than local path. 🤔 🤷🏻

**Review by navarroaxel — APPROVED** [review]

File: pages/common/pest.md (hunk 2 above)
Comment (sbrl) [reply pages/common/pest.md:6]: I agree. `current directory` makes more sense.

**Review by sbrl — APPROVED** [review]
Thanks, @owenvoke!

**Pull request merged**